pkg/kubectl/rolling_updater.go

```diff
@@ -554,7 +554,7 @@ func Rename(c coreclient.ReplicationControllersGetter, rc *api.ReplicationContro
 	if err != nil && !errors.IsNotFound(err) {
 		return err
 	}
-	err = wait.Poll(5*time.Second, 60*time.Second, func() (bool, error) {
+	err = wait.Poll(5*time.Second, 90*time.Second, func() (bool, error) {
```

Comment:
Honestly I wonder why we're not using `config.Interval` and `config.Timeout` that are used somewhere else in this code. But since this is already deprecated approach, I just want to fix the test and not to worry more about it. Not quite sure when this is expected to be entirely removed, though.